data/human/human missions.txt

```diff
@@ -6120,3 +6120,80 @@ mission "Blind Man from Martini"
 				clear "Blind Man from Martini: label far"
 				clear "Blind Man from Martini: label looking"
 			`	With that, he grabs his cane and his luggage, and follows a hospital attendant toward his daughter's room, leaving you in the reception.`
+
+mission "Republic Navy Advisory System [frozen]"
+	name "Cold Relief"
+	description "Humanitarian aid of <cargo> to <destination> following a severe cold snap."
+	minor
+	repeat
+	source
+		government "Republic"
+	destination
+		government "Republic" "Syndicate" "Independent" "Neutral"
+		attributes "frozen"
+	substitutions
+		"<problem>" "cold snap"
+		"<problem>" "freeze"
+			"random" < 60
+		"<problem>" "snowstorm"
+			"random" < 30
+	cargo "cold aid" 10 15 0.5
+	to offer
+		random < 1
+	on offer
+		dialog `This is the Republic Navy Advisory System: <planet> is experiencing a severe <problem>. Volunteers are requested to supply <cargo> to <planet>. A stipend of <payment> is provided to cover fuel, crew, and any incidental expenses.`
+	on complete
+		payment 8000
+		dialog `Shivering, yet appreciative aid workers welcome your shipment. You look on, feeling satisfied that <cargo> will soon arrive to those that need it most.`
+
+mission "Republic Navy Advisory System [volcanic]"
+	name "Volcanic Relief"
+	description "Humanitarian aid of <cargo> to <destination> following a severe volcanic eruption."
+	minor
+	repeat
+	source
+		government "Republic"
+	destination
+		government "Republic" "Syndicate" "Independent" "Neutral"
+		attributes "volcanic" "lava"
+	substitutions
+		"<problem>" "volcanic event"
+		"<problem>" "volcanic eruption"
+			"random" < 60
+		"<problem>" "gas eruption"
+			"random" < 30
+	cargo "cold aid" 10 15 0.5
+	to offer
+		random < 1
+	on offer
+		dialog `This is the Republic Navy Advisory System: <planet> is experiencing a severe <problem>. Volunteers are requested to supply <cargo> to <planet>. A stipend of <payment> is provided to cover fuel, crew, and any incidental expenses.`
+	on complete
+		payment 8000
+		dialog `Relieved-looking aid workers welcome your shipment. You look on, feeling satisfied that <cargo> will soon arrive to those that need it most.`
+
+mission "Republic Navy Advisory System [wildfire]"
```

Comment:
Two things:
- There is already a wildfire job, but we can either remove that one (because I certainly like linking it to the Navy) or limit it only to the Free Worlds. You can look at that job's payment and use it here.
- These missions would be better in `jobs.txt`.

Reply:
Will do. Also, looking at them now, I think I should probably consolidate "volcanic aid" and "wildfire aid" commodities into single "flames aid".  Their contents are basically identical.